packages/tss-lib/babel.config.js

```diff
@@ -1 +0,0 @@
-module.exports = require("@toruslabs/config/babel.config");
```

Comment:
Think this is still necessary. Please double check.

Reply:
I'm not sure. All the testing I did was fine without this. Maybe @himanshuchawla009 knows?
(I prefer to keep the configuration minimal, because more configuration also can mean more room for error.)

Reply:
had another look at this.
i don't think it is needed.

`torus-scripts` automatically uses the default babel config if none is provided. See the [README](https://github.com/torusresearch/torus-cli/tree/main/packages/torus-scripts).

Reply:
Ok, resolving this comment and related comments as well.